### PR TITLE
Miquelromero/add articles slider

### LINKED ISSUE #13: Add a row of articles in home pages
Try to reuse code from articles list

### PR DESCRIPTION
**Pull request by miquelromero**
Fixes #13

### CODE REVIEW

File: components/articles/ArticleItem.vue [hunk 1]
```diff
@@ -21,35 +21,39 @@
...
+      <v-card-actions>
+        <v-btn
+          :style="{ visibility: loaded ? 'visible' : 'hidden' }"
+          block
+          :href="url"
+          target="_blank"
+          text
+          color="primary"
+        >
+          Continua llegint
```
Comment (mverdaguer) [inline components/articles/ArticleItem.vue]: Maybe we should add a task in order to add locales to criticat.

Comment (miquelromero) [reply components/articles/ArticleItem.vue]: Absolutely

Comment (miquelromero) [reply components/articles/ArticleItem.vue]: #16

File: components/sections/slider/SliderSectionShowMore.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      Show more
+    </nuxt-link>
+    <v-speed-dial
+      v-else
+      v-model="speedDial"
+      class="show-more-buttons"
+      direction="bottom"
+      transition="slide-y-reverse-transition"
+    >
+      <template v-slot:activator>
```
Comment (MarcPuigdemont) [inline components/sections/slider/SliderSectionShowMore.vue]: otherwise, #activator for consistency

Comment (miquelromero) [reply components/sections/slider/SliderSectionShowMore.vue]: Still getting used to the shorthand

**Pull request merged**